website/docs/tutorials/08-cairo-1-support/05-compiling.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: Compilation
+---
+
+# Compiling a Cairo 1 Project with Protostar
+
+To compile your Starknet contracts written in Cairo 1 using Protostar, follow these steps:
+
+1. [Prepare your contracts](./01-understanding-cairo-packages.md).
+2. Specify the contracts in the [`protostar.toml` configuration file](/docs/tutorials/configuration-file).
+Each Cairo file that contains an entrypoint function should be listed in the [`contracts` section of the configuration file](/docs/tutorials/configuration-file#contracts).
+An entrypoint is a function decorated with `#[external]` or `#[view]`.
+3. Run the [`protostar build-cairo1`](../../cli-reference.md#build-cairo1) command.
+
+`build-cairo1` command outputs a class hash if the compilation is successful.
+
+## Output directory
+
+Running [`protostar build-cairo1`](../../cli-reference.md#build-cairo1) command will generate the compiled contracts in the `build` directory by default.
+You can specify a custom output directory using the [`--compiled-contracts-dir`](../../cli-reference#compiled-contracts-dir-pathbuild-1) argument.
+
+```
+$ protostar build-cairo1 --compiled-contracts-dir out
+```
+
+This will create the following files in the `out` directory:
+
+```
+protostar-project
+├── src
+│   ├── main.cairo
+│   └── proxy.cairo
+├── out
+│   ├── main.casm.json
+│   ├── main.sierra.json
+│   ├── proxy.casm.json
+│   ├── proxy.sierra.json
```

Comment:
The output also contains `<name>.class.hash` and `<name>.compiled.class.hash`

Reply:
[add info about class hash files](https://github.com/software-mansion/protostar/pull/1643/commits/a39139c8f7813ae4389536500df90f2f6f965a8d)